package.json

```diff
@@ -165,6 +165,7 @@
     "jest-transform-css": "^6.0.0",
     "js-yaml": "^4.1.0",
     "lint-staged": "^11.2.6",
+    "markdown-it": "^14.0.0",
```

Comment:
you dont use this package

Reply:
@diplodoc/transform doesn't work without it. 
`ModuleNotFoundError: Module not found: Error: Can't resolve 'markdown-it/lib/token'`

Reply:
fixed